test/built-ins/Promise/any/ctx-non-ctor.js

```diff
@@ -21,27 +21,3 @@ assert.sameValue(typeof Promise.any, 'function');
 assert.throws(TypeError, function() {
   Promise.any.call(eval);
 });
-
-assert.throws(TypeError, function() {
```

Comment:
While this change matches files with Promise.all, by convention we've been recently having a single file to non valid constructor types. While this remains correct I don't see any pros with this change here. I would not block it thou. We should avoid expanding every detail to a single file, right?